extra/run-dogstatsd.sh

```diff
@@ -24,6 +24,11 @@ else
   DISABLE_DATADOG_AGENT=1
 fi
 
+# match ignored hosts and disable agent
+if [[ $DD_IGNORE_HOST ]] && [[ $DYNO == *"$DD_IGNORE_HOST"* ]]; then
```

Comment:
What's the `*`s for?

Reply:
it's a simple way to match all, because the `$DYNO` variable also includes a random dyno instance number